tools/provision/debian.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# g++ stuff in wheezy has a bug which has been fixed in gcc 4.7.3 which conveniently is exactly 
+# the version that is not shipped in wheezy
+# See https://gcc.gnu.org/viewcvs/gcc/branches/gcc-4_7-branch/libstdc%2B%2B-v3/include/std/condition_variable?view=patch&r1=189276&r2=193528&pathrev=193528
+
+function patch_condition_variable()
+{
+	TARGET_DIR=/usr/include/c++/4.7
+	pushd $TARGET_DIR
+	
+	if [[ ! -f condition_variable.orig ]]; then		
+		sudo cp condition_variable condition_variable.orig
+		sudo patch -p5 < $FILES_DIR/debian/condition_variable.patch
```

Comment:
That might've worked, but condition_variable only bugs out on building rocksdb which isn't in this cmake harness. I'll leave this in as is for now.